File: Dart/Q16.dart
import 'dart:io';

void main() {
  print("************** Welcome to the Programmer Journey **************");
  print("Enter your name: ");
  String name = stdin.readLineSync()!;
  print("Enter your age: ");
  int age = int.parse(stdin.readLineSync()!);
  print("Enter your city: ");
  String city = stdin.readLineSync()!;

  print("\n********** User Details **********\n");
  print("Your Name is : $name");
  print("Your Age is  : $age");
  print("Your City is : $city");
}

File: Basics/Java/Q16.java

import java.util.Scanner;

class Q16 {

    public static void main(String[] args) {
        System.out.println("************** Welcome to the Programmer Journey **************");
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter your name: ");
        String name = sc.nextLine();

        System.out.println("Enter your age: ");
        int age = sc.nextInt();
        sc.nextLine(); //? Going for another input

        System.out.println("Enter your city: ");
        String city = sc.nextLine();

        System.out.println("\n********** User Details **********\n");

        System.out.println("Your Name is : " + name);
        System.out.println("Your Age is  : " + age);
        System.out.println("Your City is : " + city);
    }
}
